app/routes/questionnaire.py

```diff
@@ -338,6 +338,13 @@ def get_confirmation_email_sent():
 
     email = URLParamSerializer().loads(request.args.get("email"))
 
+    try:
+        ConfirmationEmail()
+    except ConfirmationEmailLimitReached:
+        show_send_another_email_guidance = False
+    else:
+        show_send_another_email_guidance = True
+
```

Comment:
I don't think we should be creating an instance of `ConfirmationEmail` to find out if the limit is reached. If the limit is not reached the init also creates a form and loads the schema.

Reply:
I'll make it a static method and rework the init.